tests/bsim/bluetooth/host/adv/chain/tests_scripts/adv_chain.sh

```diff
@@ -8,7 +8,7 @@ source ${ZEPHYR_BASE}/tests/bsim/sh_common.source
 
 simulation_id="adv_chain"
 verbosity_level=2
-EXECUTE_TIMEOUT=10
+EXECUTE_TIMEOUT=100
```

Comment:
Since this is a real-time watchdog that we rarely hit for the actual reasons we have it, can't we just default to 300 and remove all the script specific ones below 300?

Reply:
done. I also applied it to the rest of the bsim tests